manifests/dhcpmatch.pp

```diff
@@ -0,0 +1,15 @@
+# Create a dnsmasq dhcp match
+define dnsmasq::dhcpmatch (
+  $content,
+  $paramtag = undef,
+) {
+  include dnsmasq::params
+
+  $dnsmasq_conffile = $dnsmasq::params::dnsmasq_conffile
+
+  concat::fragment { "dnsmasq-dhcpmatch-${name}":
+    order   => '01',
```

Comment:
Done.
Also fixed the typos (I think). Sorry for that. Need to send another pull-request I guess?